src/models/user.ts

```diff
@@ -63,4 +63,36 @@ export const UserModel = {
 
 		return data;
 	},
+
+  /**
+	 * ユーザーを作成する。
+	 * @param {string} display_id 表示ID
+	 * @param {Object} updatedData 更新するユーザ情報
+	 * @returns {Promise<User>} 更新したユーザー
+	 * @throws {Error} DB操作に失敗した場合
+	 */
+	updateByDisplayId: async (
+		display_id: string,
+		updatedData:Partial<User>
```

Comment:
"更新データ" なので、過去形より現在形のほうが良いと思います

```suggestion
		updateData:Partial<User>
```